src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -5,32 +5,28 @@
 /**
  * All the constants to be uses for the robot
  */
-public abstract class RobotConstants extends RobotMap {
-    public LimelightConstants limelightConstants = new LimelightConstants();
-
-    public TesterConstants testerConstants = new TesterConstants();
-    public VisionConstants visionConstants = new VisionConstants();
+public class RobotConstants extends RobotMap {
```

Comment:
don't have it extent RobotMap. have robot map be it's stand alone thing just create an instance of RobotMap inside this class

Reply:
the structure is constants inheriting from map, map inheriting from local
then everything comes together in the constants.
Look at the swerve branch for an example.